spec/proxy_spec.lua

```diff
@@ -66,7 +66,7 @@ describe('Proxy', function()
     assert.same('function', type(proxy.post_action))
   end)
 
-  it('finds service by host', function()
+  pending('finds service by host', function()
```

Comment:
This needs to be moved to the find service policy spec.

Reply:
Done.